### PR TITLE
New package: limine-4.20230514.0.

### PR DESCRIPTION
**Pull request by mobinmob**
In this first release, `--enable-uefi-cd` is missing, because in my tests it breaks the build. I hope I can enable it in a future spkgbuild release :)

### CODE REVIEW

**Comment by Visone-Selektah** [comment]
It build for me with `--enable-all` or with the option `--enable-efi-cd` added

**Comment by mobinmob** [comment]
> It build for me with `--enable-all` or with the option `--enable-efi-cd` added

Really?
It did not for me (in a chroot). Let me try again...
I am not sure --enable-all makes much sense for venom, but I will try to see the results.

**Comment by mobinmob** [comment]
> It build for me with `--enable-all` or with the option `--enable-efi-cd` added

I added  `--enable-efi-cd ` , it works. `--enable-all `created a uefi binary for aarch64 also I see...

**Comment by lumarogit** [comment]
Hi !!!  clang and lld contains llvm as dep, you can remove it from depends to avoid redundant deps.

**Comment by mobinmob** [comment]
> Hi !!! clang and lld contains llvm as dep, you can remove it from depends to avoid redundant deps.

Thank you, done :)
